includes/Helpers/Transient.php

```diff
@@ -11,31 +11,36 @@ class Transient {
 	 *
 	 * If the site has an object-cache.php drop-in, then we can't reliably
 	 * use the transients API. We'll try to fall back to the options API.
-	 *
-	 * @return boolean
 	 */
-	public static function should_use_transients() {
+	protected static function should_use_transients(): bool {
 		require_once constant( 'ABSPATH' ) . '/wp-admin/includes/plugin.php';
 		return ! array_key_exists( 'object-cache.php', get_dropins() );
```

Comment:
@BrianHenryIE  - We should probably update the logic here. This is true for shared hosting where there is no object cache, but now that we have Bluehost Cloud that does have object caching, we should take this into account.